### PR TITLE
Autobots

### PR DESCRIPTION
**Pull request by sbmaxx**
Автоматически собираемые беты для PR в формате `https://pull-number.turboext.net`, and master branch — https://master.turboext.net

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# TurboExtensions
+# TurboExtensions 3
```
Comment (notary) [inline README.md]: удалить 3

File: server.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-const path = require('path');
-const yaml = require('js-yaml');
+const cheerio = require('cheerio');
 const { URL } = require('url');
 const chalk = require('chalk');
 const fs = require('fs');
-const postcss = require('./lib/postcss');
+const path = require('path');
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```
Comment (notary) [inline server.js]: could you move this line to the top?

File: server.js [hunk 3]
```diff
@@ -75,21 +46,51 @@ function getTurbo(req, query) {
...
         gzip: true
     });
 }
 
+app.use(express.static('public'));
+
+app.use((req, res, next) => {
+    const query = req.query.text;
+
+    // console.log(req.method);
```
Comment (notary) [inline server.js]: remove comments, please

File: pm2.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "turbo-dev",
```
Comment (notary) [inline pm2.json]: how many spaces pad in this file?

Comment (sbmaxx) [reply pm2.json]: two!

**Pull request merged**